pkg/utils/event_sender.go

```diff
@@ -51,6 +53,7 @@ func GetHTTPReq(funcName string, funcPort int, kafkaTopic, namespace, eventNames
 	if err != nil {
 		return nil, fmt.Errorf("Unable to create request %v", err)
 	}
+	req.Header.Set("User-Agent", fmt.Sprintf("kubeless/kafka-trigger-controller %s", version.Version))
```

Comment:
I would set this as `kubeless-kafka-trigger-controller/%s` (typically this field is ID/version)

Reply:
Updated